apps/cbportal/base/CBGrid/.macros/page/add_disktype/3_add_disktype.py

```diff
@@ -0,0 +1,21 @@
+from JumpScale.portal.docgenerator.popup import Popup
```

Comment:
why is this a macro and not defined in actions?
I dont see anything dynamic here.

Reply:
https://github.com/0-complexity/openvcloud/pull/1805/files#diff-f170d4d45e39377368fb3a3

Reply:
https://github.com/0-complexity/openvcloud/pull/1805/files#diff-f170d4d45e39377368fb3a398719ef78R10